src/lib/template.ts

```diff
@@ -202,14 +229,20 @@ export class Template {
  */
 export type TemplatePart = {
   type: 'node',
-  index: number
-}|{type: 'attribute', index: number, name: string, strings: string[]};
-
-export const isTemplatePartActive = (part: TemplatePart) => part.index !== -1;
+}|{type: 'attribute', name: string, strings: string[]};
 
 // Allows `document.createComment('')` to be renamed for a
 // small manual size-savings.
-export const createMarker = () => document.createComment('');
+export const createMarker = (data: string) => document.createComment(data);
```

Comment:
Can we use `document.createComment(data || '')` here so we can drop the `''` parameters at most of the `createComment` calls?

Reply:
Or `(data: string = '') => `

Reply:
Yes, a default value would be even better

Reply:
Both increase the size of the main binary (3 and 2 bytes respectively).

Reply:
That's surprising, there's like 12 `createMarker('')` calls so uncompressed the difference is 21 bytes in favour of the default value. I guess `('')` compresses really well. :)